File: data/src/main/java/com/github/gmetal/data/entity/AccountEntity.java
package com.github.gmetal.data.entity;

import com.bluelinelabs.logansquare.annotation.JsonField;
import com.bluelinelabs.logansquare.annotation.JsonObject;

@JsonObject
public class AccountEntity extends BaseMoviesDbEntity {

    @JsonField(name = "avatar")
    public GravatarEntity avatar;

    @JsonField(name = "id")
    public Integer id;

    @JsonField(name = "iso_639_1")
    public String iso6391;

    @JsonField(name = "iso_3166_1")
    public String iso31661;

    @JsonField(name = "name")
    public String name;

    @JsonField(name = "include_adult")
    public boolean includeAdult;

    @JsonField(name = "username")
    public String username;
}

File: data/src/main/java/com/github/gmetal/data/entity/AccountEntity.kt
package com.github.gmetal.data.entity

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
class AccountEntity : BaseMoviesDbEntity() {

    @SerialName("avatar")
    var avatar: GravatarEntity? = null

    @SerialName("id")
    var id: Int? = null

    @SerialName("iso_639_1")
    var iso6391: String? = null

    @SerialName("iso_3166_1")
    var iso31661: String? = null

    @SerialName("name")
    var name: String? = null

    @SerialName("include_adult")
    var includeAdult: Boolean = false

    @SerialName("username")
    var username: String? = null
}
